### PR TITLE
RENO-385: Adds Icons

### PR DESCRIPTION
**Pull request by helenvholmes**
Jira Ticket: http://jira.nypl.org/browse/RENO-385

## **This PR does the following:**
- Adds icon svgs and displays them in Emulsify.

### Front End Review:
- [ ] View [the example in Pattern Lab](http://localhost:3000/pattern-lab/public) and verify that everything that's left looks correct.
- [ ] Verify the navigation icons in the Slideshow look correct
- [ ] Compare to Figma

I didn't add modifier classes for applying colors to the icons—I wanted to ask how you've handled those in the past.

### CODE REVIEW

**Comment by helenvholmes** [comment]
Thanks @ModulesUnraveled! Looks like you've already rebased master in for me so I'm going to merge this.

@jackkim9, I'm happy to take flattening icons off your plate if need be.

**Comment by jackkim9** [comment]
Thank you @helenvholmes 

As mentioned offline, accessibility icons need to be 'small' given recent design changes in the address bar, but otherwise, this looks good to me. I'll put 'design review complete'.

File: components/_patterns/03-organisms/slideshow/slideshow.twig [hunk 1]
```diff
@@ -45,7 +45,7 @@
     {# Previous Arrow #}
     <div {{ bem('nav', (['prev']), slideshow__base_class) }}>
       {% include "@atoms/images/icons/_icon.twig" with {
-        icon_name: 'arrow',
+        icon_name: 'arrow-xsmall-thin',
```
Comment (ModulesUnraveled) [inline components/_patterns/03-organisms/slideshow/slideshow.twig]: Should the "next" arrow be updated to use this too?

Comment (helenvholmes) [reply components/_patterns/03-organisms/slideshow/slideshow.twig]: Yes—I've gone and updated this.

**Comment by wlla** [comment]
So are all these icons being represented within `img` or can they be implemented in other ways? Can we require an `alt` be added to all the icons before being used in a project? This may be meaningful like `alt="Search"` and it may be decorative like `alt=""`.
We may not know the exact wording that is appropriate for the icon until we know the context of the page so we don't necessarily want to include a fixed `alt`. We do however have some naming conventions so maybe we want to include default alts that can be changed or have visible text where these are displayed that include those. 

For the icons that are decorative, I don't think we want that in the title field, I think we'd want what's currently in the desc field added the title field and drop the desc field altogether. If we have alt as a required field on all icons: we don't need to indicate which are decorative until they are actually being used in which case this will be taken care of; which I believe is a departure from what I wrote up in the documentation

This comment is pretty stream of conscious; maybe let's talk.

**Comment by ModulesUnraveled** [comment]
I'm merging this so that we can get the theme integrated into the main site repository. I'll be making a list of things for @helenvholmes to check when she comes back, including @wlla's comment above.

**Pull request merged**